client/src/apiCalls.ts

```diff
@@ -0,0 +1,164 @@
+import { Player, GameState } from './types';
+
+import { close } from './userInput';
+import { PLAYER_ID } from './index';
+import { initScreenView, renderScreenView } from './blessed';
+import { URL } from './constants';
+import { GameError } from './error';
+require('isomorphic-fetch');
+
+export let currentState: GameState;
+
+export const startNewGame = async (numberPlayer: number) => {
+  close();
+  initScreenView();
+
+  const gameState = await pullNewGame(numberPlayer);
+  renderScreenView(gameState);
+
+  currentState = gameState;
+
+  setInterval(async () => {
+    const hasChanged = await pullGameStateChanged(currentState);
+
+    if (hasChanged) {
+      const gameState = await pullGameState();
+      renderScreenView(gameState);
+      currentState = gameState;
+    }
+  }, 5000);
+};
+
+export const pullNewGame = async (playerNumber: number): Promise<GameState> => {
+  try {
+    const response = await fetch(`${URL}/startgame`, {
+      method: 'POST',
+      body: JSON.stringify({ playerNumber }),
+      headers: { 'Content-Type': 'application/json' },
+    });
+    const gameState = (await response.json()) as GameState;
+    return gameState;
+  } catch (ex) {
+    throw new GameError("The game can't be launched");
+  }
+};
+
+export const pullGameState = async (): Promise<GameState> => {
+  try {
+    const response = await fetch(`${URL}/gamestate`);
+    const gameState: GameState = (await response.json()) as GameState;
```

Comment:
I would put the casting at the next line to make this line more readable

Reply:
corrected